### DISCUSSION TITLE
Did not find frame

### DISCUSSION THREAD

**Discussion by mrQuidome** [liodali/osm_flutter#298] [Q&A]
I got: E/FrameEvents(27925): updateAcquireFence: Did not find frame. in the console. Many of them. Is this something I can fix myself?
I get then when I zoom in and out and move on the screen

**Comment by liodali** [comment]
it's nothing log message and we don't need to fix it

**Comment by mrQuidome** [comment]
OK thnxs, I will ignore it

**Comment by mrQuidome** [comment]
[🔗 Answered by blasten](https://issuemode.com/issues/flutter/flutter/85084700#answer-79870836)

2022-05-23
The message updateAcquireFence: Did not find frame. isn't something to be worried about.
The Android team told us that this is likely a bug in HWUI and doesn't signal an error.

For the time being we should consider filtering this out from flutter run output since I agree it can be a lot of spam.
